tools/copy_inputs.sh

```diff
@@ -74,92 +91,113 @@ else
 fi
 
 # test if original simulation exists
-if [ ! -d $DA_EXPDIR_SRC/$src ]; then
+if [ ! -d $DA_EXPDIR/$src ]; then
   echo "original simulation $src does not exist in $DA_EXPDIR"
   echo "exit"
   exit 1
 fi
 
 # list of files to copy
-declare -a tocopy=("/namoptions." "/lscale.inp." "/prof.inp." "/scalar.inp." "/purifs.inp." "/trees.inp." "/scals.inp." "/lad.inp." "/facetarea.inp." "/facets.inp." "/netsw.inp." "/svf.inp." "/Tfacinit.inp." "/vf.nc.inp." "/factypes.inp.")
+declare -a tocopy=("/namoptions." "/lscale.inp." "/prof.inp." "/scalar.inp." "/purifs.inp." "/trees.inp." "/scals.inp." "/lad.inp." "/facetarea.inp." "/facets.inp." "/facets_unused." "/netsw.inp." "/svf.inp." "/Tfacinit.inp." "/vf.nc.inp." "/factypes.inp.")
```

Comment:
Also add `timedepsw.inp.`, `timedeplw.inp.`, `timedepnudge.inp.`, and `vfsparse.inp.` to this list.

Reply:
Done!